pipeline/live_pipeline.py

```diff
@@ -70,19 +71,20 @@ def update_log_date_and_log(log_date: str, log: list) -> tuple[str, list]:
 def save_log_to_file(log_date: str, log: list) -> None:
     """Saves the log to a json file"""
     json_dict = {log_date: log}
-    with open(FILE_NAME, "w", encoding="utf-8") as f:
+    with open("/tmp/" + FILE_NAME, "w", encoding="utf-8") as f:
         json.dump(json_dict, f)
 
 
 def upload_log_to_s3(aws_client: client) -> None:
     """Uploads the log file to S3"""
-    # tmp_path = "/tmp/" + FILE_NAME  # for lambda
-    tmp_path = FILE_NAME  # for local
+    tmp_path = "/tmp/" + FILE_NAME  # for lambda
+    # tmp_path = FILE_NAME  # for local
```

Comment:
Are we leaving the comments in?

Reply:
Leaving it in for now in case we needed to test writing to a file locally, will make note to remove at end of week, thanks Miles.